src/main/java/org/apache/commons/text/translate/NumericEntityUnescaper.java

```diff
@@ -143,8 +160,33 @@ public int translate(final CharSequence input, final int index, final Writer out
                 final char[] chrs = Character.toChars(entityValue);
                 out.write(chrs[0]);
                 out.write(chrs[1]);
+
+            } else if (128 <= entityValue && entityValue <= 159  // must be within the cp-1252 extension range
+                    && !isHex  // must be a NUMERIC entity, not hex entity (see StringEscapeUtilsTest for hex)
+                    && !INVALID_CP1252_POINTS.contains(entityValue)  // must not be an invalid code point for cp-1252
+            ) {
+                System.err.println(entityValue);
```

Comment:
We probably don't want to have a `System.err` in functions called by users as this may clutter their logs.

Reply:
Oh whoops that should have been removed!